app/__init__.py

```diff
@@ -1,11 +1,18 @@
 from flask import Flask
-from bu_cascade.cascade_connector import Cascade
+# from bu_cascade.cascade_connector import Cascade
 
 app = Flask(__name__)
 app.config.from_object('config')
 app.url_map.strict_slashes = False
 
-cascade_connector = Cascade(app.config['SOAP_URL'], app.config['CASCADE_LOGIN'], app.config['SITE_ID'], app.config['STAGING_DESTINATION_ID'])
+# cascade_connector = Cascade(app.config['SOAP_URL'], app.config['CASCADE_LOGIN'],
+#                             app.config['SITE_ID'], app.config['STAGING_DESTINATION_ID'])
```

Comment:
Do we not need this anymore? If we don't, we should delete it

Reply:
I think this was initially imported so we could use Cascade groups to determine who should be allowed to use this app, but then we migrated over to IAM groups via LDAP. I wasn't able to find any usages of `cascade_connector` in Cache Clear, so I just commented it out.

I agree though: if it's not used, it should be removed. I'll go do that.